core/typescript/src/lib/domain.ts

```diff
@@ -13,10 +13,29 @@
  * @group For both project creators and package authors
```

Comment:
"Project Creators" aka App Developers, there's no third user persona right? should replace. Project Creator is very  FABR context specific to understand compared to App Developer which is intuitive.

Reply:
I'm neutral on this one. On one hand, exactly as you said in order to standardise the terms we use, on the other I'm relaxed about using different sets of terms between OSS and SaaS where we talk more sharply about specific persons that require accurate messaging. 
I think we should merge this in, and do a global search-n-replace if needed later.